static/app/data/forms/accountPreferences.tsx

```diff
@@ -28,6 +28,10 @@ const formGroups: JsonFormObject[] = [
           {value: 'system', label: t('Default to system')},
         ],
         getData: transformOptions,
+        onChange: () => {
+          // Remove any existing theme class from when the page was loaded
+          document.body.classList.remove('theme-light', 'theme-dark', 'theme-system');
```

Comment:
I'd make this a helper function so we don't repeat the theme class names in more than 1 place. That said, for the current state of things I'm fine with only 2 places and 3 names especially if it is unlikely that we'd need this anywhere else. Maybe add a `TODO:` note for this kind of future work?